src/rules/at-import-no-partial-extension/index.js

```diff
@@ -23,6 +23,13 @@ export default function (on, options) {
     })
     if (!validOptions) { return }
 
+    result.warn((
```

Comment:
This one bothers me. The idea was to give users a warning that this rule is deprecated. But the downside is that all the rule's tests are failing (each is giving +1 warning).